pkg/apis/cluster/v1alpha1/imageregistry_types.go

```diff
@@ -0,0 +1,99 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ImageRegistrySpec is the spec of managedClusterImageRegistry.
+type ImageRegistrySpec struct {
+	// Registry is the address of overridden image registry
+	// +required
+	Registry string `json:"registry,omitempty"`
+
+	// PullSecret is the name of image pull secret which should be in the same namespace with the managedClusterImageRegistry.
+	// +required
+	PullSecret string `json:"pullSecret,omitempty"`
```

Comment:
we may use `corev1.LocalObjectReference` instead of the `string`, the type is used to let you locate the
referenced object inside the same namespace.

Reply:
done